deployment/ansible/roles/raster-foundry.jenkins/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# tasks for raster-foundry.jenkins
+
+- name: Add new Nginx config
```

Comment:
It may make sense to name this task `Add Jenkins Nginx config` vs. `Add new Nginx config` so that a clear connection between the task can be established by someone skimming the playbook.